spec/data/base.po

```diff
@@ -0,0 +1,38 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Base header\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+
+msgid "This little piggie is unchanged"
+msgstr "1"
+
+msgid "This little piggie is removed from remote, unchanged on local"
+msgstr "2"
+
+msgid "This little piggie is removed from remote, changed on local"
+msgstr "3"
+
+msgid "This little piggie is removed from local, unchanged on remote"
+msgstr "4"
+
+msgid "This little piggie is removed from local, changed on remote"
+msgstr "5"
+
+msgid "This little piggie is changed on local, unchanged on remote"
+msgstr "6"
+
+msgid "This little piggie is changed on remote, unchanged on local"
+msgstr "7"
+
+msgid "This little piggie is changed on remote and local"
+msgstr "8"
+
+# msgid "This little piggie is added on remote, not on local"
```

Comment:
It's commented on purpose — that way all 3 test files have the same "content", with some "added" (uncommented) and some "removed" (commented).